Gemfile

```diff
@@ -49,7 +49,7 @@ gem "linux_admin",                    "~>2.0", ">=2.0.1", :require => false
 gem "listen",                         "~>3.2",         :require => false
 gem "log_decorator",                  "~>0.1",         :require => false
 gem "manageiq-api-client",            "~>0.3.4",       :require => false
-gem "manageiq-loggers",               "~>0.5.0",       :require => false
+gem "manageiq-loggers",               "~>0.6",         :require => false
```

Comment:
I'm tempted to either lock down to `"~> 0.6.0"` or release a v1.0 and `"~> 1.0"`.  Anyone else have any thoughts?

Reply:
I can lock down in this PR if you'd prefer.

Reply:
Ok I locked it to 0.6.0